### ISSUE TITLE
[chore] Attempt to link straight to same doc in different version

### ISSUE THREAD

**Issue by andrewnicols** [moodle/devdocs#477]
### What do we need to do?

Regarding versioned documentation, a suggestion from Eloy:

> If you hover the menu (or click on the "triangle") it correctly shows you the available options (master, 4.1 and legacy). Then can select any of them, all ok. But, if you click on the "name" of the current version, then you're redirected to the "base" document in that version. IMO, that's completely unexpected. The "name" should behave the very same than the "triangle" or become 100% disabled.

**Comment by andrewnicols** [comment]
It looks like we already do this...

<img width="1624" alt="Screen Shot 2022-12-12 at 12 54 14" src="https://user-images.githubusercontent.com/370047/206963644-986caf79-6ca1-49cc-a5b1-38334a2713e9.png">

Where are you seeing this _not_ happen?

**Comment by stronk7** [comment]
Yep, yep, clicking on the hovered drop-down elements does send you to the correct page, so you can switch between the "Advanced Grading API" versions perfectly. That works ok.

The issue I reported was that clicking on the drop-down (name or triangle), sends you to the "base" page. And that's the behaviour I'm not sure it's 100% nice. Maybe it's only a matter of getting used for the hover to appear, but my first impulse was to go straight and click on the "name + triangle" (I didn't know that the menu was going to show automatically).

Not sure if I'm explaining it properly, heh. Just in any versioned page click on the top name or triangle. Do you think the current action (go to "base" page) is ok? As said maybe it's only a matter of getting used, just was not "natural" for me.

Ciao :-)

**Comment by andrewnicols** [comment]
Ah thank s- I got it. I'll see what we can do
